src/commands/org/list.ts

```diff
@@ -8,17 +8,28 @@
 import { Flags, loglevel, SfCommand } from '@salesforce/sf-plugins-core';
 import { AuthInfo, ConfigAggregator, ConfigInfo, Connection, Org, SfError, Messages, Logger } from '@salesforce/core';
 import { Interfaces } from '@oclif/core';
+import * as chalk from 'chalk';
 import { OrgListUtil, identifyActiveOrgByStatus } from '../../shared/orgListUtil';
 import { getStyledObject } from '../../shared/orgHighlighter';
 import { ExtendedAuthFields, FullyPopulatedScratchOrgFields } from '../../shared/orgTypes';
 
 Messages.importMessagesDirectory(__dirname);
 const messages = Messages.loadMessages('@salesforce/plugin-org', 'list');
 
+const defaultOrgEmoji = '🍁';
+const defaultHubEmoji = '🌳';
+
 export type OrgListResult = {
+  /**
+   * @deprecated
+   * preserved for backward json compatibility.  Duplicates devHubs, sandboxes, regularOrgs, which should be preferred*/
   nonScratchOrgs: ExtendedAuthFields[];
   scratchOrgs: FullyPopulatedScratchOrgFields[];
+  sandboxes: ExtendedAuthFields[];
+  regularOrgs: ExtendedAuthFields[];
```

Comment:
what are "regular" orgs?

Reply:
not scratch, sandbox, or devHubs.  got a better name?